src/components-examples/material/table/table-http/table-http-example.ts

```diff
@@ -16,7 +16,7 @@ import {catchError, map, startWith, switchMap} from 'rxjs/operators';
 export class TableHttpExample implements AfterViewInit {
   displayedColumns: string[] = ['created', 'state', 'number', 'title'];
   exampleDatabase: ExampleHttpDatabase | null;
-  data: GithubIssue[] = [];
+  githubIssuesObservable: Observable<GithubIssue[]>;
```

Comment:
Not sure that we should be suffixing observables with `Observable`.

Reply:
Yeah, I just wanted something readable and understandable for new Angular developers. Just `githubIssues` or `data` lacks context that could help newer Angular devs.

I really dislike Hungarian notation and I don't think that we use it in this project?

Reply:
OK, I think that I found a descriptive name that doesn't include `Observable`.